docs/customizing.md

```diff
@@ -0,0 +1,33 @@
+# Customizing Licensee's behavior
+
+## Adjusting the confidence threshold
+
+If you'd like, you can make Licensee less stringent in its comparison, but risk getting false positives as a result. The confidence threshold is an integer between 1 and 100, with the default being 95, meaning that License is at least 95% confident that the file represents the matched license.
+
+```ruby
+LICENSEE.confidence_threshold
+=> 95
+
+LICENSEE.confidence_threshold = 90
+=> 90
+```
+
+## Matching package manager metadata
+
+Licensee supports the ability to take into account Ruby or Node package manager metadata, disabled by default. [There are reasons you may not want to use this metadata](what-we-look-at). You can explicitly instruct licensee to take this information into account as follows:
+
+```ruby
+project = Licensee::Project.new("path/to/project", detect_packages: true)
+project.license
+=> #<Licensee::Licensee key="mit">
+```
+
+## Matching project README license references
```

Comment:
This sentence would be easier to read if reordered in SVO: "Matching license references in project READMEs"